Makefile.development.mk

```diff
@@ -18,8 +13,7 @@ endif
 template: ## Output the rendered Helm template
 	$(eval CHART_DIR := "helm/cluster")
 	$(eval HELM_RELEASE_NAME := $(shell yq .global.metadata.name ${CHART_DIR}/${CI_FILE}))
-	@cd ${CHART_DIR} && \
-		helm template -f ${CI_FILE} --debug ${HELM_RELEASE_NAME} .
+	@helm template ${HELM_RELEASE_NAME} ${CHART_DIR} --values ${CI_FILE} --debug
```

Comment:
This breaks `make template`

```
make template
install.go:214: [debug] Original chart version: ""
install.go:231: [debug] CHART PATH: /Users/nikola/src/giantswarm/cluster/helm/cluster

Error: open ci/ci-values.yaml: no such file or directory
helm.go:84: [debug] open ci/ci-values.yaml: no such file or directory
make: *** [template] Error 1
```

Reply:
Ah, yeah, my mistake! Need to prepend the "${CHART_DIR}"